skimage/transform/radon_transform.py

```diff
@@ -23,10 +9,10 @@
 from warnings import warn
 
 
-__all__ = ["radon", "iradon", "iradon_sart"]
+__all__ = ['radon', 'order_angles_golden_ratio', 'iradon', 'iradon_sart']
```

Comment:
Having looked at the docstring, to my admittedly-naive eyes, I agree that it's worthwhile exposing it.